src/build123d/geometry.py

```diff
@@ -1376,149 +1421,80 @@ class Location:
         Extrinsic.ZYZ: gp_EulerSequence.gp_Extrinsic_ZYZ,
     }
 
-    @overload
-    def __init__(self):  # pragma: no cover
-        """Empty location with not rotation or translation with respect to the original location."""
-
-    @overload
-    def __init__(self, location: Location):  # pragma: no cover
-        """Location with another given location."""
-
-    @overload
-    def __init__(self, translation: VectorLike, angle: float = 0):  # pragma: no cover
-        """Location with translation with respect to the original location.
-        If angle != 0 then the location includes a rotation around z-axis by angle"""
-
-    @overload
-    def __init__(
-        self, translation: VectorLike, rotation: RotationLike | None = None
-    ):  # pragma: no cover
-        """Location with translation with respect to the original location.
-        If rotation is not None then the location includes the rotation (see also Rotation class)
-        """
-
     @overload
     def __init__(
         self,
-        translation: VectorLike,
-        rotation: RotationLike,
-        ordering: Extrinsic | Intrinsic,
-    ):  # pragma: no cover
-        """Location with translation with respect to the original location.
-        If rotation is not None then the location includes the rotation (see also Rotation class)
-        ordering defaults to Intrinsic.XYZ, but can also be set to Extrinsic
+        position: VectorLike | None = None,
```

Comment:
Why `None` here?  What should happen if the user passed `None` as all three parameters?

Reply:
I was thinking this captures the empty init case as well, `Location()` -> `Location((0,0,0),(0,0,0))`

Reply:
RE: what happens when user passes None as all three parameters, this is what happens currently.
```py
Location(position=None, orientation=None, ordering=None) # Returns: Location((0,0,0),(0,0,0))
Location(None, None, None) # Returns: TypeError: Unexpected input type: [<class 'NoneType'>, <class 'NoneType'>, <class 'NoneType'>]